resources/config-defaults.edn

```diff
@@ -103,5 +103,8 @@
  ;; :disable-commands [:application.command/assign-external-id :application.command/change-resources]
  :disable-commands []
 
+ ;; List of allowed values for the organization field. Should contain at least one element.
+ :organizations [""]
```

Comment:
After thinking a bit more, I'm not sure how "" will work as a default without thinking a few more steps ahead than I'm currently capable of. But I guess this is fine for now and we can change it later on as needed.

Reply:
after discussion, changed it to `["default"]`